behave/docker-compose.yml

```diff
@@ -39,6 +39,10 @@ services:
       CLA_PROVIDER_SECRET_ID: 0494287c65bdf61d29f0eeed467ec8e090f0d80f
 
   seleniumchrome:
+    deploy:
```

Comment:
This will help to make things run locally with the same resources constraints on all of our machines, otherwise we would have a situations where things work for people with powerful machines and not working in circle or other people machines locally